docs/source/method_cards/methods_overview.rst

```diff
@@ -18,61 +17,54 @@
    .. link-button:: augmix.html
       :text: AugMix
       :classes: card-title
-
    :badge:`CV,badge-primary`
    ++++++++++++++
-   Image-preserving data augmentations
+   Image-perserving data augmentations
```

Comment:
should be preserving?

Reply:
Ah - it's grabbed from the docstring, I think, so will update that shortly